### DISCUSSION TITLE
Enable tasks

### DISCUSSION THREAD

**Discussion by komir2004** [akeeba/panopticon#302] [Q&A (Community Support)] [answered]
Hi, I'm trying to figure tasks on the page. I see that all the tasks are enabled except for "extensionsupdate". I manually enable it and it turns green. But when I reload the page, it goes back to gray. 
Is this a bug or am I doing something wrong?

**Comment by nikosdion** [comment, marked as answer]
You are not supposed to manage tasks manually. That page is there in case I need to give you instructions to delete a stuck task, or if you want to see what tasks are currently running and scheduled.

**Comment by komir2004** [comment]
Thank you for clarifying
